docs/governance/README.md

```diff
@@ -0,0 +1,127 @@
+  Standard a given library should depend on. It also makes it easier for .NET
+  implementations to decide whether it's worth implementing a higher version of
+  .NET Standard.
+* **The version number** is subject to discussion and is generally a function of
+  how significant the new version is. While we aren't planning on making
+  breaking changes, we'll rev the major version if the new version adds large
+  chunks of APIs or has sizable changes in the overall developer experience.
+
+[netstandard-api]: https://github.com/dotnet/standard/issues?q=is%3Aopen+is%3Aissue+label%3Anetstandard-api
+
+## Inclusion principles
+
+Not every .NET API needs to be (or even should be) part of the .NET Standard.
+
+If a library is written in pure IL (for example C#, VB.NET, F# etc.), and
+targets .NET Standard, then it can run across all current and future .NET
+implementations with no code changes. The only requirement on the .NET
+implementation is that it has to support at least the version of .NET Standard
+that the library was compiled for (or a higher version).
+
+Any modifications to such a library, whether they're bug fixes, performance
+improvements, or additional APIs, will also work across all these .NET
+implementations as well. That means the library author has flexibility to make
+releases on their own cadence while consumers of the library have the choice on
+when they upgrade the version they are using. If the library is made a part of
+.NET Standard, it can only version at the same cadence as the standard, so the
+update flexibility is much reduced.
+
+On the other hand, there are also downsides for APIs not being part of the
+standard. If an API ships as a library, it's much harder for any .NET
+implementation to use the types as part of their implementation or public API
+surface. Hence, there is a lot of value in adding widely used concepts to the
+.NET Standard in order to use them throughout all .NET implementations.
+
+The following criteria helps us to identify APIs that should be part of .NET
+Standard:
+
+* **Ubiquitous APIs**. APIs in the .NET Standard must be implemented by all .NET
+  implementations. Thus, we're only interested in standardizing APIs that are
+  universal in nature and thus should be available everywhere. Of course, this
+  only includes .NET implementations that are still updated.
+
+* **Mature APIs**. APIs that are part of the .NET Standard can only be versioned
+  when the standard itself is versioned. Thus, we generally only standardize
+  APIs that are mature enough that they don't have to be updated frequently.
+  This implies that we will never add APIs that aren't stable yet.
+
+* **Runtime-specific APIs**. If the APIs have to be implemented by the runtime,
+  they don't benefit from being libraries on top of .NET Standard. Good examples
+  of such APIs include primitive types, reflection, GC, and hardware intrinsics.
+
+* **Widely-used APIs**. In order to enable a vibrant .NET ecosystem, it's
```

Comment:
I think it might be worth calling out some counter examples of APIs that we wouldn't include. Things for new technologies that are not yet industry standards.

Reply:
I thought we covered this under "mature APIs".

Reply:
Partially. I still think there could be debate about whether or not an API is mature vs a Technology is mature. I'm OK either way.

Reply:
That's fair. To me, it's hard to envision a world where the API is mature but the tech isn't though. The closes thing I can think of app model like technologies (like DI containers or OR mappers) which seem to be subject to fashion and patterns which don't tend to last for very long. I don't know how to phrase that though. Maybe a new bullet point like this:

> **Fundamental**. The standard is about foundational technologies. We want to avoid adding concepts that are often subject to changes, such as DI containers or OR mappers.

Reply:
Wouldn't, generally, something like DI or OR be "on" netstandard vs "in" netstandard?

Reply:
Asside: While I know you are trying to call out what should be in the standard or not. I think labeling and calling out the difference between "on" and "in" would be a big help and I can also have a place to point folks at whenever I have to explain that concept.

Reply:
Alright, I've added section before and discussed the "on" vs "in" a lot more. Let me know if this helps.